misc/net.minetest.minetest.metainfo.xml

```diff
@@ -25,11 +25,30 @@
         <display_length compare="ge">360</display_length>
     </requires>
 
-    <content_rating type="oars-1.0">
+    <content_rating type="oars-1.1">
         <content_attribute id="violence-cartoon">mild</content_attribute>
         <content_attribute id="violence-fantasy">mild</content_attribute>
+        <content_attribute id="violence-realistic">none</content_attribute>
+        <content_attribute id="violence-bloodshed">none</content_attribute>
+        <content_attribute id="violence-sexual">none</content_attribute>
+        <content_attribute id="violence-desecration">none</content_attribute>
+        <content_attribute id="violence-slavery">none</content_attribute>
+        <content_attribute id="drugs-alcohol">none</content_attribute>
+        <content_attribute id="drugs-narcotics">none</content_attribute>
+        <content_attribute id="drugs-tobacco">none</content_attribute>
+        <content_attribute id="sex-nudity">none</content_attribute>
+        <content_attribute id="sex-themes">none</content_attribute>
+        <content_attribute id="language-profanity">none</content_attribute>
+        <content_attribute id="language-humor">none</content_attribute>
+        <content_attribute id="language-discrimination">none</content_attribute>
```

Comment:
how do we correctly answer these? ContentDB mods/games could include any of this